patches/server/1047-Add-UUID-attribute-modifier-API.patch

```diff
@@ -0,0 +1,32 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TonytheMacaroni <tonythemacaroni123@gmail.com>
+Date: Thu, 9 Nov 2023 20:34:44 -0500
+Subject: [PATCH] Add UUID attribute modifier API
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/attribute/CraftAttributeInstance.java b/src/main/java/org/bukkit/craftbukkit/attribute/CraftAttributeInstance.java
+index 0029412b1242879deb898524001bb4cc7550fa78..42917d79e20163a44f41c77f8a3d5e2472bb7758 100644
+--- a/src/main/java/org/bukkit/craftbukkit/attribute/CraftAttributeInstance.java
++++ b/src/main/java/org/bukkit/craftbukkit/attribute/CraftAttributeInstance.java
+@@ -44,6 +44,21 @@ public class CraftAttributeInstance implements AttributeInstance {
+         return result;
+     }
+ 
++    // Paper start
++    @Override
++    public AttributeModifier getModifier(java.util.UUID uuid) {
++        Preconditions.checkArgument(uuid != null, "uuid");
```

Comment:
This error message is not helpful, same below

Reply:
Changed the error message.